base/Dockerfile

```diff
@@ -47,6 +50,17 @@ RUN apt-get update && \
         curl -s -S -o - "${KOPANO_CORE_REPOSITORY_URL}/Release.key" | apt-key add -; \
     fi
 
+# Create kopano user and group
+RUN groupadd -r --gid ${KOPANO_GID} kopano
+RUN useradd -r -s /usr/sbin/nologin -d /var/lib/kopano -g ${KOPANO_GID} -u ${KOPANO_UID} kopano
```

Comment:
here I am also leaning slightly toward the long options. then you could also break up the command into multiple lines. but that would be more of a cosmetic nature.

Reply:
Actually this is the current user creation part from the postinst from kopano-common:

```
useradd -r -c 'Kopano Groupware Suite' -g kopano -d /dev/null \
        -s /sbin/nologin kopano
```

maybe the home dir should also be set to /dev/null?

Reply:
> maybe the home dir should also be set to /dev/null?

What is the purpose of setting the home directoy to /dev/null (which is not really a directory...) when we use a nologin shell?

Reply:
No real purpose, other than replicating existing behaviour.